ReactWindows/ReactNative.Net46/Modules/Storage/AsyncStorageModule.cs

```diff
@@ -286,17 +283,19 @@ public void OnDestroy()
         private async Task<string> GetAsync(string key)
         {
             var storageFolder = await GetAsyncStorageFolder(false).ConfigureAwait(false);
-            if (storageFolder != null)
+            if (storageFolder == null)
+            {
+                return null;
+            }
+
+            var fileName = AsyncStorageHelpers.GetFileName(key);
+            if (storageFolder.CheckExistsAsync(fileName).Result == ExistenceCheckResult.FileExists)
```

Comment:
Yes please, that was definitely an oversight.

---
In reply to: [99771671](https://github.com/ReactWindows/react-native-windows/pull/967#discussion_r99771671) [](ancestors = 99771671)